master/usage/decommissioning-a-node.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Decommissioning a Node
+---
+
+If you are decommissioning a host that runs calico/node (e.g. because it
+is being taken out of service, or is being renamed) it is necessary to
+explicitly delete the associated [Node resource][Node resource reference]
+to remove any data associated with that node.  In environments where nodes
+will be added and removed regularly this clean up should be included in the
+cluster management system. This page provides instruction on how to remove
+a host's associated node data.
+
+> **Notes**
+>
+> - Removing a Node resource will also remove the Workload Endpoint
```

Comment:
Should we only focus on WorkloadEnpoints here?  I think host protection and the IP addresses are also important, e.g.

Removing a Node resource will remove all of the Calico data associated with that Node.  This includes Workload Endpoints, Host Endpoints and IP addresses assigned by Calico IPAM.